Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1080 @@
+                    )
+
+                elif lumu_status in ["close", "closed"]:
+                    msg = "to close an incident, it has to be in the CLOSE INCIDENT option in the ACTION button"
+                    response = {"statusCode": 404, "reason": msg}
+                    demisto.debug(msg)
+
+                elif original_commnet:
+                    new_comment = generate_hmac_sha256_msg(key, new_comment)
+                    response = client.comment_a_specific_incident_request(
+                        new_incident_id, new_comment
+                    )
+
+                else:
+                    response = {"statusCode": 404, "reason": "lumu status not found"}
+
+            elif original_commnet:
+
+                new_comment = generate_hmac_sha256_msg(key, new_comment)
+
+                response = client.comment_a_specific_incident_request(
+                    new_incident_id, new_comment
+                )
+
+        except DemistoException as err:
+            response = repr(err)
+            error_msg = "Failed to parse json object from response: b''"
+            if str(err) == error_msg:
+                response = {"statusCode": 200}
+            else:
+                response = repr(err)
+                raise DemistoException(err)
+
+    demisto.debug(
+        f"{response=}, {parsed_args.delta=}, {parsed_args.remote_incident_id=}, {parsed_args.incident_changed}"
+    )
+
+    return new_incident_id
+
+
+def clear_cache_command():
+    cache: Dict = {}
+    cache["cache"] = []
+    cache["lumu_incidentsId"] = []
+    set_integration_context(cache)
+    return f"cache cleared {get_integration_context()=}"
+
+
+def get_cache_command():
+
+    cache = get_integration_context()
+
+    # demisto.debug(f'{cache=}')
+
+    command_results = CommandResults(
+        outputs_prefix="Lumu.GetCache",
+        outputs_key_field="cache",
+        outputs=cache,
+        raw_response=cache,
+    )
+
+    return command_results
+
+
+def test_module(client: Client, args: Dict[str, Any]) -> None:
+    # Test functions here
+    try:
+        page = args.get("page")
+        items = args.get("items")
+        client.retrieve_labels_request(page, items)
+        return_results("ok")
+    except Exception as e:
+        msg_err = f"verify Lumu API Key and Network Connections - {e.__class__} - {repr(e)} - {e}"
+        return_results(msg_err)
+
+
+def main() -> None:
+
+    params: Dict[str, Any] = demisto.params()
+    args: Dict[str, Any] = demisto.args()
+    url = params.get("url")
+    verify_certificate: bool = not params.get("insecure", False)
+    proxy = params.get("proxy", False)
+    offset_set_off = params.get("fetch_offset")
+
+    headers: Dict = {}
+
+    command = demisto.command()
+    demisto.debug(f"Command being called is {command.upper():*^40}")
+
+    try:
+        urllib3.disable_warnings()
+        client: Client = Client(
+            urljoin(url, ""), verify_certificate, proxy, headers=headers, auth=None
```

Comment:
- why using `urljoin(url, "")` ?
- no need to pass empty auth and headers
```suggestion
            urljoin(url, ""), verify_certificate, proxy
```

Reply:
solved it in commit 2bb633df7de2b53b93d8a8cdc1fcaef13e33c6a7

Reply:
why you need this line? 
`urljoin(url, "")`

Reply:
removed urljoin